doc/en/getting-started.rst

```diff
@@ -77,6 +77,13 @@ This test returns a failure report because ``func(3)`` does not return ``5``.
 
     You can use the ``assert`` statement to verify test expectations. pytest’s `Advanced assertion introspection <http://docs.python.org/reference/simple_stmts.html#the-assert-statement>`_ will intelligently report intermediate values of the assert expression so you can avoid the many names `of JUnit legacy methods <http://docs.python.org/library/unittest.html#test-cases>`_.
 
+.. note::
```

Comment:
Following the discussion, how about we replace line 74 by:

```
The ``[100%]`` refer to the overall progress of running all test cases. After it finishes, pytest then shows a failure report because ``func(3)`` does not return ``5``.
```

And remove the note? What do you think @katrinleinweber?

Reply:
Thanks for the suggestion! LGTM